config.js

```diff
@@ -32,6 +33,7 @@ if (fs.existsSync(jsonPath)) {
 		database: env('DATABASE', 'mongodb://localhost/pa11y-webservice'),
 		host: env('HOST', '0.0.0.0'),
 		port: Number(env('PORT', '3000')),
+		runners: env('RUNNERS', ['htmlcs', 'axe']),
```

Comment:
Would this not need to use `possibleCsvListToArray`? Additionally we should not default to using both `htmlcs` and `axe` as this will break peoples dashboards who upgrade but don't add this configuration.

```suggestion
		runners: possibleCsvListToArray(env('RUNNERS', 'htmlcs')),
```